### PR TITLE
Add Burst Balance metric for AWS EC2 EBS plugin

### PR DESCRIPTION
**Pull request by ariarijp**

### CODE REVIEW

File: mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go [hunk 1]
```diff
@@ -92,6 +96,10 @@ var cloudwatchdefs = map[string](cloudWatchSetting){
 		MetricName: "VolumeConsumedReadWriteOps", Statistics: "Sum",
 		CalcFunc: func(val float64, period float64) float64 { return val },
 	},
+	"ec2.ebs.burst_balance.#.burst_balance": cloudWatchSetting{
+		MetricName: "BurstBalance", Statistics: "Average",
```
Comment (astj) [inline mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go]: ref: http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ebs-metricscollected.html

File: mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go [hunk 2]
```diff
@@ -261,7 +276,9 @@ func (p EBSPlugin) FetchMetrics() (map[string]interface{}, error) {
 		volumeID := normalizeVolumeID(*vol.VolumeId)
 		graphs := defaultGraphs
 		if *vol.VolumeType == "io1" {
-			graphs = allGraphs
+			graphs = io1Graphs
+		} else if *vol.VolumeType == "gp2" {
```
Comment (astj) [inline mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go]: According to the Cloudwatch reference, BurstBalance metric is also available on st1 and sc1 volumes.
Please support them also.

**Comment by astj** [comment]
@ariarijp Thank you for the pull request! reviewed.

File: mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go [hunk 3]
```diff
@@ -31,10 +31,14 @@ var defaultGraphs = []string{
 	"ec2.ebs.idle_time.#",
 }
 
-var allGraphs = append([]string{
+var defaultGraphs = append([]string{
+	"ec2.ebs.burst_balance.#",
```
Comment (ariarijp) [inline mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go]: It is used for gp2, st1 and sc1 storage.

File: mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go (hunk 2 above)
Comment (ariarijp) [inline mackerel-plugin-aws-ec2-ebs/lib/aws-ec2-ebs.go]: @astj Thank you for your advice!
I have pushed the code changes.

**Comment by astj** [comment]
Thank you! merged.

**Pull request merged**